content/en/monitors/guide/division-in-monitor-evaluations.md

```diff
@@ -0,0 +1,211 @@
+
+## Error Rate Example
+
+Suppose you want to monitor an error rate over 5 minutes using the metrics, `requests.error` and `requests.total`. There are 2 possible ways to perform this calculation.
+
+Consider a single evaluation performed with these aligned timeseries points for the 5 min timeframe:
+
+**Numerator**: `sum:requests.error{*}`
+
+```
+| Timestamp             | Value       |
+| :-------------------- | :---------- |
+| 2018-03-13 11:00:30   | 1           |
+| 2018-03-13 11:01:30   | 2           |
+| 2018-03-13 11:02:40   | 3           |
+| 2018-03-13 11:03:30   | 4           |
+| 2018-03-13 11:04:40   | 5           |
+```
+
+**Denominator**: `sum:requests.total{*}`
+
+```
+| Timestamp             | Value    |
+| :-------------------- | :------- |
+| 2018-03-13 11:00:30   | 5        |
+| 2018-03-13 11:01:30   | 5        |
+| 2018-03-13 11:02:40   | 5        |
+| 2018-03-13 11:03:30   | 5        |
+| 2018-03-13 11:04:40   | 5        |
+```
+
+### Division with `as_count()`
+
+Monitors involving division and at least 1 `as_count()` modifier use a separate evaluation path than other monitors. This impacts queries of the format `queryA.as_count() / queryB.as_count()`. Queries without division (`+`, `-`, or `*`) are not impacted.
+
+Refer this query **`as_count_eval_path`**:
+
+```
+sum(last_5m): sum:requests.error{*}.as_count() / sum:requests.total{*}.as_count()
+```
+
+and this query **`classic_eval_path`**\*:
+
+```
+sum(last_5m): sum:requests.error{*}.as_rate() / sum:requests.total{*}.as_rate()
+```
+
+Compare the result of the evaluation depending of the path:
+
+| Path                     | Behavior                                       | Expanded expression         | Result  |
+| :----------------------- | :--------------------------------------------- | :-------------------------- | :------ |
+| **`as_count_eval_path`** | Aggregation function applied _before_ division | **(1+2+...+5)/(5+5+...+5)** | **0.6** |
+| **`classic_eval_path`**  | Aggregation function applied _after_ division  | **(1/5 + 2/5 + ... + 5/5)** | **3**   |
+
+You could also use `avg` and `as_rate()` to get the same result as `sum` and `as_count()` :
+
+```
+avg(last_5m): sum:requests.error{*}.as_rate() / sum:requests.total{*}.as_rate()
+```
+
+| Path                    | Behavior                                      | Expanded expression           | Result  |
+| :---------------------- | :-------------------------------------------- | :---------------------------- | :------ |
+| **`classic_eval_path`** | Aggregation function applied _after_ division | **(1/5 + 2/5 + ... + 5/5)/5** | **0.6** |
+
+_Note that all evaluations above are mathematically correct. Choose a method to suit your intention._
+
+\*If the metric is a rate, you can optionally omit `.as_rate()`. Include this modifier to be explicit.
```

Comment:
For both `COUNT` and `RATE` type metrics, either `.as_count()` or `.as_rate()` is required for the query to evaluate correctly. The UI query editor will maintain this invariant for you, but if you're creating a monitor via the API you need to add it to the query manually.

Reply:
Actually this isn't completely accurate. You can construct a valid query without either of these modifiers, but we don't support doing that. I'll add more details in another comment below.